### PR TITLE
Removing Unused Arguments

### PR DESCRIPTION
**Pull request by nabobalis**
**Updated version of #1690**

These unused arguments to methods were listed under Minor Issues.

### CODE REVIEW

**Comment by Cadair** [comment]
I think I am 👎 on this, many of these changes seem to be counter intuitive.

If some of these keyword arguments are not used then they should be removed altogether not folded into `*args` or `**kwargs`.

**Comment by nabobalis** [comment]
I did so.

**Comment by nabobalis** [comment]
That is one way to screw up a rebase.

**Comment by nabobalis** [comment]
@Cadair I think its ready.

File: sunpy/conftest.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def pytest_runtest_teardown(item, nextitem):
     disable_internet.turn_on_internet()
 
 
-def pytest_unconfigure(config):
+def pytest_unconfigure():
```
Comment (Cadair) [inline sunpy/conftest.py]: Doesn't pytest expect this to take one argument?

Comment (nabobalis) [inline sunpy/conftest.py]: Yes you are right.

File: sunpy/lightcurve/sources/rhessi.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ class RHESSISummaryLightCurve(LightCurve):
     * Mission Paper `<http://link.springer.com/article/10.1023%2FA%3A1022428818870>`_
     """
 
-    def peek(self, title="RHESSI Observing Summary Count Rate", **kwargs):
+    def peek(self):
```
Comment (dpshelio) [inline sunpy/lightcurve/sources/rhessi.py]: what's wrong with this default title for rhessi? Is it coming from the data? should the title description in the docstring also be removed?

Comment (Cadair) [inline sunpy/lightcurve/sources/rhessi.py]: there is nothing wrong with the string, it's just not used in the function anywhere.

Comment (nabobalis) [inline sunpy/lightcurve/sources/rhessi.py]: Changed the doc string.

**Comment by nabobalis** [comment]
Since this is partly affecting #2116. I might just merge this. @Cadair

**Pull request merged**